package.json

```diff
@@ -30,7 +30,8 @@
     "access": "public"
   },
   "dependencies": {
-    "core-js-bundle": "^3.0.0-beta.13"
+    "core-js-bundle": "^3.0.0-beta.13",
```

Comment:
Where do we need the core-js? In the changes here i don't see it at the first look?

Reply:
Inside the main.js. And  they have good stable versions. I think, we could update it.

Reply:
Btw, It is marked as a change, because a comma was added, because a new element has been added to this json-structure. :)